lib/aerosol/launch_configuration.rb

```diff
@@ -90,6 +92,12 @@ def to_s
 }}
   end
 
+  def corrected_user_data
+    realized_user_data = user_data.is_a?(Proc) ? user_data.call : user_data
```

Comment:
I'm not sure you'd gain anything.  From within the block, you have access to the `meta_data` method already which will return that data, and you also have access to the rest of the object.  Passing the argument really doesn't give you anything other than two ways to access the same variable.